src/chat/ChatScreen.tsx

```diff
@@ -96,6 +98,16 @@ export const ChatScreen: React.FC<ChatScreenProps> = ({
     [firebaseInstance, memberIds, partners]
   );
 
+  const onLoadEarlier = useCallback(async () => {
+    if (conversationRef.current?.id) {
+      const res = await firebaseInstance.getMoreMessage();
+      setHasMoreMessages(res.length > 0);
```

Comment:
Why has more message when res.length > 0?

Reply:
because we need disable loadEarlier={hasMoreMessages}, if not loading indicator will show forever

Reply:
if you call `getMoreMessage` and it return only 1 message, should we show load more?

Reply:
No, **onLoadEarlier** function only called when message res.length >= 20 at line 65, So when go to the first time will not show any load more. Only when more then 20 messages and scroll to top, **onLoadEarlier** will trigger and load more

Reply:
what happen if we have 21 messages?

Reply:
**hasMoreMessages** is true and **onLoadEarlier** depend on **loadEarlier** function is true. So it still working well. This is video demo when 21 messages:

https://github.com/saigontechnology/rn-firebase-chat/assets/105196859/139f6de0-d859-4d32-9928-1b10f6b24078

Reply:
I addressed check duplicate, please help me verify it again. Thank you